### DISCUSSION TITLE
Database schema requirements

### DISCUSSION THREAD

**Discussion by vamsibvv** [rahulvemula/SE-leasingPortal#38] [Q&A] [answered]
Let us discuss here what all are the tables required and what are the attributes that we are going to capture in the UI. 

We will be creating these tables in the dbfree site so that everyone can use them appropriately.

**Comment by mitulmandaliya** [comment, marked as answer]
Issue #6 mentions the dbi tables

**Reply by vamsibvv** [reply]
Great, We can proceed accordingly!

**Comment by rahulvemula** [comment]
Tables needed for the initial sprint sorted from high priority to low priority:

TABLE 1:
Society table:
societyId - primary
societyName
societyAddress
societyCity
societyAmenities - arr
societyImg

TABLE 2:
Listing table:
lisitingId
listingType - enum (0 house 1 bedroom)
houseType - enum (1/2/3/4 bhk)
Rent
societyId - foreign key to soc table
isLeased - boolean
listingImg
sqft
listingLabel - (Name of the listing. Ex: private bedroom in 3bhk)

TABLE 3:
User table:
userID
userName
firstName
lastName
phoneNumber

TABLE 4:
Lease table:
leaseId - primary
lisitingId - foreign
userID - foreign
leaseStartDate
leaseEndDate

This should suffice for the initial API development for listSocieties, listListings. Additional columns can be added to these tables according to requirement when the new stories are picked up.
